modules/identityserver/test/Volo.Abp.IdentityServer.TestBase/Volo/Abp/IdentityServer/AbpIdentityServerTestBaseModule.cs

```diff
@@ -12,6 +12,18 @@ namespace Volo.Abp.IdentityServer
     )]
     public class AbpIdentityServerTestBaseModule : AbpModule
     {
+        public override void PreConfigureServices(ServiceConfigurationContext context)
+        {
+            PreConfigure<AbpIdentityServerBuilderOptions>(options =>
+            {
+                options.AddDeveloperSigningCredential = false;
+            });
+
+            PreConfigure<IIdentityServerBuilder>(identityServerBuilder =>
+            {
+                identityServerBuilder.AddDeveloperSigningCredential(false);

```

Comment:
When the `AddDeveloperSigningCredential `method finds a `tempkey.rsa`  file in the directory, it tries to use it.
We can specify a file name to prevent the previous `tempkey.rsa` from being used.
This is why I use Guid as the file name. 

https://github.com/IdentityServer/IdentityServer4/blob/4ef0886e03d2b10acdbd2e876f521d6b636fc81d/src/IdentityServer4/src/Configuration/DependencyInjection/BuilderExtensions/Crypto.cs#L175

Reply:
Well I thought it's unnecessery io check for file existence since it will be clean/cleaned project but maybe caution is better this case.
I'll update.